app/resources/services/integrations/coil.js

```diff
@@ -0,0 +1,167 @@
+const config = require('config')
+const logger = require('../../../../lib/logger.js')()
+const btoa = require('btoa')
+const { User } = require('../../../db/models')
+const passport = require('passport')
+const OAuth2Strategy = require('passport-oauth').OAuth2Strategy
+const axios = require('axios')
```

Comment:
Let's re-sort imports so that modules (like 'config' or 'passport') are imported first, and other scripts are loaded second. We don't enforce this rule but you can see it described here: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md (and we should probably adopt this lint rule at some point)

Reply:
I think we should just implement this as an eslint rule then.